server/s3_delete_object_action.h

```diff
@@ -138,8 +141,10 @@ class S3DeleteObjectAction : public S3ObjectAction {
   FRIEND_TEST(S3DeleteObjectActionTest, MarkOIDSForDeletion);
   FRIEND_TEST(S3DeleteObjectActionTest, DeleteObjectsDelayedEnabled);
   FRIEND_TEST(S3DeleteObjectActionTest, DeleteObjectsDelayedDisabled);
-  FRIEND_TEST(S3DeleteObjectActionTest, DeleteHandlerInEnabledState);
+  FRIEND_TEST(S3DeleteObjectActionTest, DeleteHandlerCreateDeleteMarker);
   FRIEND_TEST(S3DeleteObjectActionTest, MetadataHandlerInEnabledState);
+  FRIEND_TEST(S3DeleteObjectActionTest, DeleteHandlerDeleteDeleteMarker);
+  FRIEND_TEST(S3DeleteObjectActionTest, DeleteVersionMetadata);
 };
 
 #endif
```

Comment:
Any ST cases required for this changes?

Reply:
ST will be added after delete first and last index available
as we can not delete bucket
added in separate ticket